community/event-checklist.md

```diff
@@ -0,0 +1,95 @@
+toc=true
+
++++
+
+# 1. Logistics - Before the Workshop
+
+✔️ Decide the **_date and time_** of the event as per the convenience of speakers and audience. Conducting **_events during exams_** or some other important event might **_not be a good idea_** in general. Checking **_academic calendar_** and other events in KGP is suggested.
+
+✔️ Publicize the event at least **_2 or 3_** days before the workshop. Publicizing earlier will fade away the event info from memory and publicizing late will be difficult for the audience to plan for.
+
+✔️ Share the Post on Facebook, Twitter and Linkedin via KOSS Account.
+
+✔️ Speakers should be **_decided atleast a week_** before the workshop, giving them enough time to prepare.
+
+✔️ It is important to have _atleast two_ backup speakers for each speaker.(One for the speaker, one for the backup speaker)
+
+✔️ Speakers of the event should have enough practical experience with the topic. The experience is necessary so that speaker can convey the content in the simplest way and clear doubts.
+
+✔️ It is highly recommended that the Slides for the event are made from scratch by the speaker.  Preparing from scratch will add creativity and newness to the content. New and simpler ways to present a topic might come out.  Also...
+
+​      🗸 Feel free to look up at previous slides at [kossiitkgp/slides](https://github.com/kossiitkgp/slides) repo for inspiration.
+
+​      🗸 Make sure that something important that has been covered in the past doesn't gets removed.
+
+​      🗸 Feel free to remove or edit stuff if you ever feel some part of the content is unnecessary and irrelevant, after discussing with team mates. 
+
+✔️ Make sure to take feedback on the slides(content) from other members of KOSS. It is suggested to have the discussion over a meet to discuss things like potential queries from audience, thinking of ways to simplify content.
+
+✔️ Refer [kossiitkgp/mistakes](https://github.com/kossiitkgp/mistakes) repository to get an idea of what sought of things can go wrong.
+
+✔️ It is a good idea to record the event(offline/online) for the sake of posterity.
+
+✔️ Make a Github repo for the workshop with name as event name and year which contains - installation instructions, slides, references, recording and others. (For example refer - [1](https://github.com/kossiitkgp/git-and-github-workshop-2020), [2](https://github.com/kossiitkgp/REST-APIs-in-Flask-Workshop-2021), [3](https://github.com/kossiitkgp/Linux-Shell-Vim-Workshop-2021))
+
```

Comment:
One thing which I used to do is that I would message the people who had shown interest in the topic being presented by asking me questions on it before the workshop was planned and oftentimes they did show up.

Reply:
The main aim of the repo was
- Audience can get all the information about the workshop like installation, slides and references at any time they want.
- Future KOSS members can know how the workshop was conducted and can have all the material for reference.


> message the people who had shown interest in the topic being presented

This is a nice idea. But messaging people will restrict. What was done for the recent workshops was that - we use to put the meeting link in the common group (of executives and CTMs) and interested people would join. And we would discuss slide by slide and see where potentially audience get confused and also each member presents his ideas to simplify things. 

This worked really well for the recent workshops - Linux, REST API workshop.  Hence I put this point in the doc, which covers it
![image](https://user-images.githubusercontent.com/44907757/117695744-2fef4c80-b1de-11eb-9bf8-e21a722b8e86.png)